package.json

```diff
@@ -79,13 +79,33 @@
     "request": "^2.81.0",
     "semver": "^5.3.0",
     "simple-git": "^1.65.0",
-    "sinon": "^2.2.0",
+    "sinon": "^7.1.1",
     "tarjan-graph": "^0.3.0",
     "tmp": "0.0.26",
     "upath": "^0.1.7",
     "uuid4": "^1.0.0",
     "when": "^3.7.7",
-    "wreck": "^12.2.2"
+    "wreck": "^12.2.2",
+    "npm-which": "^3.0.1",
+    "object-to-spawn-args": "^1.1.1",
+    "conventional-changelog-cli": "^1.3.2",
+    "gulp": "github:gulpjs/gulp",
+    "gulp-angular-gettext": "2.1.0",
+    "gulp-autoprefixer": "3.1.0",
+    "gulp-bump": "2.0.1",
+    "gulp-dedupe": "0.0.2",
+    "gulp-eslint": "^4.0.0",
+    "gulp-exec": "2.1.2",
+    "gulp-git": "2.5.2",
+    "gulp-git-streamed": "2.4.0",
+    "gulp-if": "2.0.0",
+    "gulp-livereload": "3.8.1",
+    "gulp-prompt": "0.1.2",
+    "gulp-rename": "1.2.2",
+    "gulp-sass": "4.0.1",
+    "gulp-sass-lint": "1.1.1",
+    "gulp-util": "3.0.7",
+    "jest": "^21.2.1"
```

Comment:
This npm added to make jest command available, so we can use in the tests a command without any meaning